package.json

```diff
@@ -59,6 +61,7 @@
     "import:arcturus": "node --optimize_for_size --max-old-space-size=64 index.js",
     "start": "npm-run-all --parallel server webpack-dev browser-sync",
     "start:frontend": "FRONTEND=1 npm-run-all --parallel webpack-dev browser-sync:frontend",
+    "start:frontend:proxy": "FRONTEND=2 npm-run-all --parallel webpack-dev browser-sync:frontend",
```

Comment:
oh, yeah, I might have added CORS to my reverse proxy config when I needed that first command... you can replace that first command by what you implemented